packages/definitions/dist/fhir/r4/profiles-medplum.json

```diff
@@ -749,14 +749,72 @@
           {
             "id" : "ProjectMembership.accessPolicy",
             "path" : "ProjectMembership.accessPolicy",
-            "definition" : "The access policy for the user within the project memebership.",
+            "definition" : "The access policy for the user within the project memebership. If the \"access\" field is specified, then the \"accessPolicy\" field is ignored.",
```

Comment:
Should we mark this as "deprecated", to nudge users towards the new system?

Reply:
@rahul1 

Good question.  I don't know...  It might still be valuable as a simple default?

It has never been exposed via API, so it would not be a breaking change to remove it entirely.

I don't have strong opinions on that right now.